ersilia/cli/commands/fetch.py

```diff
@@ -105,7 +105,7 @@ def fetch(
             fg="blue",
         )
         mf = ModelFetcher(
-            repo_path=repo_path,
+            from_dir=repo_path or from_dir,
```

Comment:
I would not do this. I would keep the repo_path parameter in the `ModelFetcher` class, and set it with the value of `from_dir`.